data-structures/Heap.cpp

```diff
@@ -0,0 +1,84 @@
+#include<iostream>
+#include<vector>
+using namespace std;
+
+class Heap{
```

Comment:
Use this code style conventions for class-formatting https://google.github.io/styleguide/cppguide.html#Class_Format